api/commands/run.ts

```diff
@@ -145,10 +145,10 @@ export class Runner {
     }
   }
 
-  private executeNode(node: dataform.IExecutionNode) {
+  private async executeNode(node: dataform.IExecutionNode) {
     const startTime = process.hrtime();
     // This creates a promise chain that executes all tasks in order.
-    node.tasks
+    await node.tasks
```

Comment:
revert this, and the change to `executeNode` (it doesn't need to be `async` once this is removed)